cmd2/cmd2.py

```diff
@@ -5335,9 +5335,12 @@ def async_alert(self, alert_msg: str, new_prompt: Optional[str] = None) -> None:
             if update_terminal:
                 import shutil
 
+                # Prior to Python 3.11, this can return 0 for things like pseudo terminals.
+                terminal_columns = shutil.get_terminal_size().columns or 80
```

Comment:
Weird, I never considered the possibility of a zero-width terminal, but a guys for a pty that might be semi-reasonable.

nit: Consider defining a single constant for something like `DEFAULT_TERMINAL_WIDTH = 80`